tests/Doctrine/Tests/DBAL/Functional/StatementTest.php

```diff
@@ -50,6 +54,75 @@ public function testStatementIsReusableAfterClosingCursor(): void
         self::assertEquals(2, $id);
     }
 
+    public function testFetchAllWithOneArgument(): void
+    {
+        $table = new Table('stmt_fetch_all');
+        $table->addColumn('a', 'integer');
+        $table->addColumn('b', 'integer');
+        $this->connection->getSchemaManager()->createTable($table);
+
+        $this->connection->insert('stmt_fetch_all', [
+            'a' => 1,
+            'b' => 2,
+        ]);
+
+        $stmt = $this->connection->prepare('SELECT a, b FROM stmt_fetch_all');
+        $stmt->execute();
+
+        self::assertEquals([[1, 2]], $stmt->fetchAll(FetchMode::NUMERIC));
+    }
+
+    /**
+     * @depends testFetchAllWithOneArgument
+     */
+    public function testFetchAllWithTwoArguments(): void
+    {
+        $driver = $this->connection->getDriver();
+
+        if ($driver instanceof IBMDB2Driver) {
+            $this->markTestSkipped('Not supported on IBM DB2');
+        }
+
+        if ($driver instanceof MysqliDriver) {
+            $this->markTestSkipped('Not supported on Mysqli');
+        }
+
+        if ($driver instanceof OCI8Driver) {
+            $this->markTestSkipped('Not supported on OCI8');
+        }
+
+        if ($driver instanceof PDOOracleDriver) {
+            $this->markTestSkipped('Not supported on PDO Oracle');
```

Comment:
Why not? I suspect that the test may fail because Oracle by default upper-cases the column names but it's not not supported. This is one of the reasons to not use this feature.

Reply:
Indeed [it fails](https://github.com/doctrine/dbal/runs/904835107) with:

> Failed asserting that null matches expected 1.

What's the problem with simply skipping PDO Oracle?

Reply:
The misleading message. This functionality is supported on `pdo_oci`.

Reply:
True. What do you suggest?

Reply:
Given that you'll need to use an external PDO connection, you may try to use `PDO::ATTR_CASE` on the connection you're passing to make it convert all column names to the lower case as the portability wrapper does.